core/src/main/java/com/datastax/oss/driver/internal/core/channel/DefaultWriteCoalescer.java

```diff
@@ -31,6 +30,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 import net.jcip.annotations.ThreadSafe;
+import org.jctools.queues.atomic.MpscLinkedAtomicQueue;
```

Comment:
Needed because netty shade's jctools and shaded packages are not automatically exported by maven-bundle-plugin, so just added jctools as a dependency, although maybe its better to shade it ourselves since its a tiny dependency.

Reply:
Yeah, I think it would be a good idea to shade it, both in the regular and in the Netty-shaded core.
The package should be exported, in case a client of the shaded core writes their own flusher.

Reply:
Sounds good, i'll add that as it's pretty easy to do.